src/simularium/BinaryFileReader.ts

```diff
@@ -263,7 +263,17 @@ export default class BinaryFileReader implements ISimulariumFile {
         return frameContents;
     }
 
+    getSpatialData(): DataView {
+        return this.spatialDataBlock;
+    }
+
     getPlotData(): Plot[] {
         return this.plotData;
     }
+
+    getAsBlob(): Blob {
+        return new Blob([this.fileContents], {
+            type: "text/plain;charset=utf-8",
+        });
```

Comment:
I would have thought this should not be using "text/plain" with utf-8...?

Reply:
Maybe this should be "application/json;charset=utf-8" instead of "text/plain;charset=utf-8"". This came from my code originally, and you're right it's probably not the right MIME type although it seemed to generate valid output with this constructor set up.

Reply:
oh I was going completely the other way in thinking it should be a binary mime type for BinaryFileReader.  As long as it works and the file can be reloaded, i'm good!